tt_metal/hw/firmware/src/slave_idle_erisc.cc

```diff
@@ -54,9 +54,7 @@ int main(int argc, char *argv[]) {
     conditionally_disable_l1_cache();
     DIRTY_STACK_MEMORY();
     WAYPOINT("I");
-
-    int32_t num_words = ((uint)__ldm_data_end - (uint)__ldm_data_start) >> 2;
-    l1_to_local_mem_copy((uint *)__ldm_data_start, (uint tt_l1_ptr *)MEM_SLAVE_IERISC_INIT_LOCAL_L1_BASE_SCRATCH, num_words);
+    do_crt1((uint32_t *)MEM_IERISC_INIT_LOCAL_L1_BASE_SCRATCH);
```

Comment:
this should be `MEM_SLAVE_IERISC_STACK_BASE` 

also we recently made a change to `blackhole_140_arch.yaml` to comment out eth cores because we were debugging some issues with new FW on CI machines. Could you uncomment this and run 

`TT_METAL_SLOW_DISPATCH_MODE=1 ./build/test/tt_metal/unit_tests --gtest_filter=BlackholeSingleCardFixture.*`

Reply:
good catch. Out of the 3 tests, first passes, the other 2 fail with `cannot map elf file into memory`

Reply:
let me know how to reproduce the elf mapping failure and I'll take a look

Reply:
just try to run the test Almeet mentioned above after enabling the eth cores in `blackhole_140_arch.yaml` 
(line 38)

Reply:
has to be done on a BH machine available on IRD